app/services/hyrax/workflow/abstract_notification.rb

```diff
@@ -29,6 +30,15 @@ def message
           "#{title} (#{work_id}) was advanced in the workflow by #{user.user_key} and is awaiting approval #{comment}"
         end
 
+        def document
```

Comment:
@jcoyne technically it is protected so it does not need yard doc, but I can certainly add some.